helm/drone-fork-approval-extension/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Why would this chart require ingress? Is there a scenario where drone isn't on kubernetes but the plugin is?

Reply:
Not that I can think of.  We certainly wouldn't use it that way.  I'll pull this out.

Reply:
Removed in 636981a.